src/Schema/Domain/Content/Worker/FieldDefinition/AddFieldDefinitionToDomainContentMutation.php

```diff
@@ -24,33 +24,40 @@ class AddFieldDefinitionToDomainContentMutation extends BaseWorker implements Wo
      */
     private $typesInputMap;
 
-    public function __construct($typesInputMap = [])
+    /**
+     * @var \EzSystems\EzPlatformGraphQL\Schema\Domain\Content\Mapper\FieldDefinition\FieldDefinitionInputMapper[]
+     */
+    private $mappers;
+
+    /**
+     * @param string[] $typesInputMap
+     * @param \EzSystems\EzPlatformGraphQL\Schema\Domain\Content\Mapper\FieldDefinition\FieldDefinitionInputMapper[] $mappers
+     */
+    public function __construct(array $typesInputMap = [], array $mappers = [])
     {
         $this->typesInputMap = $typesInputMap;
+        $this->mappers = $mappers;
     }
 
     public function work(Builder $schema, array $args)
     {
-        $fieldDefinition = $args['FieldDefinition'];
-        $contentType = $args['ContentType'];
-
-        $fieldDefinitionField = $this->getFieldDefinitionField($fieldDefinition);
+        $properties = ['description' => $args['FieldDefinition']->getDescriptions()['eng-GB'] ?? ''];
```

Comment:
You can't really expect that `eng-GB` translation will be available. We don't force any particular naming convention for language identifiers. Consider:
* using main language from Content Type
* using first element from `getDescriptions()` array

Reply:
Yeah... well, if you look at the rest of the code, this issue exists in many places :)

But for this one, I'll change it to use the main language, good tip, thank you.

Reply:
d8f61b535f590cdebe127dd43f96e1d5c2027658. Better, @webhdx ?